### PR TITLE
Add open source tools and tutorials

### PR DESCRIPTION
**Pull request by akegaviar**
Added a bunch of tools and tutorials mostly coming from Chainstack

### CODE REVIEW

**Comment by KenMan79** [comment]
> Added a bunch of tools and tutorials mostly coming from Chainstack

@akegaviar cool, I'm eager to check it out.

File: README.md [hunk 1]
```diff
@@ -87,6 +87,8 @@ Contributions to this list are wery welcome :dolphin: You can add links through
 - [Quorum-Splunk](https://www.splunk.com/en_us/blog/it/introducing-splunk-app-for-quorum.html) - Using Splunk for quorum blockchain analytics and dashboards
 - [Mirror Besu](https://www.npmjs.com/package/mirror-besu) - compiling, testing and deployment for Hyperledger Besu, aiming to make life as an Enterprise Ethereum developer easier.
 - [Ethernal](https://tryethernal.com/) - Block explorer for private EVM-based chains.
+- [Quorum-toolbox](https://github.com/chainstack/quorum-toolbox) - Tools and scripts for Quorum configuration.
+- [Web3Quorum](https://github.com/chainstack/web3quorum) - A web3py extension that supports Quorum APIs: Raft and Istanbul.
```
Comment (nmvalera) [inline README.md]: Should this be listed under section libraries next to other web3 libraries? Thanks!

**Review by nmvalera — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (akegaviar) [reply README.md]: Sure, done.

**Review by nmvalera — APPROVED** [review]
LGTM.

Thanks a lot.

**Pull request merged**